components/form/FormItem/index.tsx

```diff
@@ -279,24 +279,28 @@ function InternalFormItem<Values = any>(props: FormItemProps<Values>): React.Rea
           "`shouldUpdate` and `dependencies` shouldn't be used together. See https://ant.design/components/form/#dependencies.",
         );
         if (Array.isArray(children) && hasName) {
-          warning(false, 'Form.Item', '`children` is array of render props cannot have `name`.');
+          warning(
+            false,
+            'Form.Item',
+            'A `Form.Item` with a `name` prop must have a single child element. For information on how to render more complex form items, see https://ant.design/components/form/#components-form-demo-complex-form-control.'
```

Comment:
Try https://github.com/ant-design/u.ant.design

Reply:
Cool, I've opened a PR to add a couple links for these error messages, I'll update this PR when that is merged 👍